extras/mobile-view.rst

```diff
@@ -0,0 +1,156 @@
+﻿Mobile View
+===========
+
+The need for a dedicated mobile view of Zammad arose from the ever-changing life
+on the go. Even though the desktop application might be responsive enough for
+small screens, it proved to be too cluttered and was simply not designed
+primarily for mobile devices.
+
+By limiting the amount of information to only the most important, the mobile
+view strives to provide you, the user, with a more focused window into your
+daily tasks. All packaged in a touch-friendly and modern design with great user
+experience, of course!
+
+.. list-table::
+
+   * - .. figure:: /images/extras/mobile-view/mobile-view-login.png
+          :alt: Mobile View Login Screen
+          :align: center
+          :width: 100%
+
+          Login Screen
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-home.png
+          :alt: Mobile View Home Page
+          :align: center
+          :width: 100%
+
+          Home Page
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-ticket-create.png
+          :alt: Mobile View Ticket Create Screen
+          :align: center
+          :width: 100%
+
+          Ticket Create Screen
+
+   * - .. figure:: /images/extras/mobile-view/mobile-view-ticket-detail-view.png
+          :alt: Mobile View Ticket Detail View
+          :align: center
+          :width: 100%
+
+          Ticket Detail View
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-article-reply.png
+          :alt: Mobile View Article Reply Dialog
+          :align: center
+          :width: 100%
+
+          Article Reply Dialog
+
+     - .. figure:: /images/extras/mobile-view/mobile-view-organization.png
+          :alt: Mobile View Ticket Organization Tab
+          :align: center
+          :width: 100%
+
+          Ticket Organization Tab
```

Comment:
Is there a specific reason why you use tables in that scope?
Depending on the readers screen size the screenshots are virtually unreadable.

This could be solved by reducing the number of columns from either 3 to 2 or by using e.g. the tab functionality of restructured text which is available in this documentation as well:


```
... tabs::

   .. tab:: One

      Content one

   .. tab:: Two

      Content two

```